### DISCUSSION TITLE
Auth0 + web3Auth allowed origins

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#545] [Help]
I want to use the auth0 + web3Auth. In Auth0, after creating a "Single Page Application (SPA)" project, there are 3 fields in the settings page, "Allowed Callback Url", "Allowed Logout Urls", "Allowed WEb Origins".

I will be using the mainnet in my verifier. So what urls do I need to put in these fields?

The only thing that I know right now is that in "Allowed Callback Urls", I need to put this Url: https://app.openlogin.com/auth and in "Allowed Logout Urls" I need to put the url where I want my user to redirect after the user is logged out. Apart from this, what exactly I need to enter?

**Comment by shahbaz17** [comment]
Hey @KetulCodiste 

These fields are sufficient to work with Auth0 x Web3Auth.

Are you facing any issues/errors?

**Reply by KetulCodiste** [reply]
Allowed Callback Url:  https://app.openlogin.com/auth
Allowed Logout Urls: https://mywebsite.com
Allowed Web Origins:  https://app.openlogin.com/auth

I need to know, that apart from these, do I need to make any kind of changes in these 3 fields?

**Reply by yashovardhan** [reply]
Hey Ketul,

As mentioned in this guide, you only need to add these callback URLs. https://web3auth.io/docs/guides/auth0#setup

Try to login with it and if there's an error let us know.
